bin/cargo-bolero/src/build_clusterfuzz.rs

```diff
@@ -0,0 +1,102 @@
+use crate::{list::List, project::Project};
+use anyhow::{Context, Result};
+use std::{
+    collections::{hash_map::DefaultHasher, HashMap},
+    convert::TryInto,
+    hash::{Hash, Hasher},
+    path::{Path, PathBuf},
+};
+use structopt::StructOpt;
+
+const OUTPUT_DIR: &str = "target/fuzz";
+
+/// Builds a tarball for uploading to clusterfuzz
+#[derive(Debug, StructOpt)]
+pub struct BuildClusterfuzz {
```

Comment:
Could you add an integration test that shows this all works how we would expect? I'm not familiar with clusterfuzz and don't want to have an accidental regression in behavior in the future.

Reply:
I think I’ve added a reasonable-ish integration test in the cargo_bolero.rs file.

Hopefully I have now handled all your review comments! Please let me know what you think about the new code, and the integration test that gives an example of how clusterfuzz runs the fuzzers :)